pkg/cmd/pr/status/status.go

```diff
@@ -316,6 +316,6 @@ func printPrs(io *iostreams.IOStreams, totalCount int, prs ...api.PullRequest) {
 	}
 	remaining := totalCount - len(prs)
 	if remaining > 0 {
-		fmt.Fprintf(w, cs.Gray("  And %d more\n"), remaining)
+		fmt.Fprintf(w, cs.Muted("  And %d more\n"), remaining)
```

Comment:
How about using `Mutedf` and `Fprintln`? One small difference is that `\n` will be out of the muted formatting, so the color-reset escape code happens at the same line and the new line starts clean.

```suggestion
		fmt.Fprintln(w, cs.Mutedf("  And %d more", remaining))
```